clients/java/.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think these can be auto-generated by a Gradle plugin: https://docs.gradle.org/current/userguide/eclipse_plugin.html

Maybe we should configure that and exclude this and the `.project` file from Git